index.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="main.css" />
+    <title>Singolo</title>
+  </head>
+  <body id="home">
+
+    <div class="contain_header">
+      <header>
+        <div class="hamburger-menu">
+          <input id="menu__toggle" type="checkbox" />

```

Comment:
Where is block "menu"? Rules of BEM are used incorrectly over all html. You need read theory once again.